README.md

```diff
@@ -152,3 +152,23 @@ export { Incoming as default };
 ```
 
 
+### SES Incoming Event
+
+/streams/table-name.mjs
+
+```
+
+class TableName {
+  
+  // do something and then finish by resolving res promise
+  constructor(req, res) {
```

Comment:
It would be good if we can use the eventName to call a function on the class like we do in APIGateway. Something like 

```
class TableName {
  
  insert(req, res) {

  }  
  
  delete(req, res) {

  }

}